File: Algorithms/selection_sort.py
list = [55, 4, 2, 17, 1, 3]

for i in range(6):
	minimum = i
	for j in range(i, 6):
		if list[minimum] > list[j]:
			minimum = j
	list[i], list[minimum] = list[minimum], list[i]

print(list)

File: C/selection_sort.c
#include <stdio.h>

void main(){
	int arr[] = {55, 4, 2, 17, 1, 3};
	for(int i =0; i<6; i++){
		int minimum = i;
		for (int j = i; j < 6; j++){
			if(arr[minimum] > arr[j]){
				minimum = j;
			}
		}
		int x = arr[i];
		int y = arr[minimum];
		
		arr[i] = y;
		arr[minimum] = x;
	}
	
	for(int i = 0; i<6; i++){
		printf("%d ", arr[i]);
	}
}